easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.0.6-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'BerkeleyGW'
+version = "1.0.6"
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+                 properties and the optical responses of a large variety of materials from bulk
+                 periodic crystals to nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.berkeleygw.org/releases/']
+sources = ['BGW-%(version)s.tar.gz']
+checksums = ['8740562da41e87a62eb8d457675e4bfd']
+
+patches = ['BerkeleyGW_ignore_archmk.patch']
+
+files_to_copy = ['*']
+
+prebuildopts = 'COMPFLAG=-DINTEL PARAFLAG=-DMPI MATHFLAG=-DUSESCALAPACK FCPP="cpp -ansi" F90free="$MPIF90 -free" '
+prebuildopts += 'LINK="$MPIF90" FOPTS="$FFLAGS" MOD_OPT="-module " INCFLAG="-I" C_PARAFLAG=-DPARA CC_COMP="$MPICXX" '
+prebuildopts += 'C_COMP="$MPICC" C_LINK="$MPICXX" C_OPTS="$CFLAGS" REMOVE="/bin/rm -f" MKLPATH="$MKLROOT" '
+prebuildopts += 'FFTWLIB="$MKLROOT/lib/intel64/libfftw2xf_double_intel.a" FFTWINCLUDE="$FFTW_INC_DIR" '
```

Comment:
You can check with `--extended-dry-run` or `-x` to see how `$LIBFFT` will be defined, but it's good to do that on a system where the toolchain you're using is actually available (since the definition depends on which libraries are actually present on the filesystem):

```
$ eb --from-pr 2925 -x | grep LIBFFT
  export LIBFFT="-Wl,-Bstatic -Wl,--start-group -lfftw3xc_intel -lfftw3x_cdft_lp64 -lmkl_cdft_core -lmkl_blacs_intelmpi_lp64 -lmkl_intel_lp64 -lmkl_sequential -lmkl_core -Wl,--end-group -Wl,-Bdynamic"
```

Note that these are the FFTW 3.x wrappers...

If this is not sufficient/correct, then you can't use it obviously, it just came to mind.